io.catenax.batch/3.0.0/Batch.ttl

```diff
@@ -0,0 +1,194 @@
+#######################################################################
+# Copyright(c) 2024 BASF SE
```

Comment:
I think your old models were created in 2022 and 2023. so it should be 2022-2024